src/main/java/net/imagej/patcher/LegacyExtensions.java

```diff
@@ -319,10 +319,18 @@ private static void insertRefreshMenusHook(CodeHacker hacker) {
 	private static void addEditorExtensionPoints(final CodeHacker hacker) {
 		hacker.insertAtTopOfMethod("ij.io.Opener", "public void open(java.lang.String path)",
 			"if (isText($1) && ij.IJ._hooks.openInEditor($1)) return;");
-		hacker.dontReturnOnNull("ij.plugin.frame.Recorder", "void createMacro()");
-		hacker.replaceCallInMethod("ij.plugin.frame.Recorder", "void createMacro()",
-			"ij.IJ", "runPlugIn",
-			"$_ = null;");
+		if (LegacyInjector.isImageJ1VersionAtLeast(hacker, "1.53g")) {
+			// Don't let ImageJ call "new Editor(...)".
+			hacker.replaceCallInMethod("ij.plugin.frame.Recorder", "void createMacro()",
+				"ij.plugin.frame.Editor", "<init>",
```

Comment:
For future reference, `"<init>"` was the method "name" we were looking for in order to match `new Editor()`.

Although this says `<clinit>`...

https://www.javassist.org/html/javassist/CtConstructor.html#getName()

Let me double check this.

Reply:
I remember now. The new call is parsed as a [`NewExpr`](https://www.javassist.org/html/javassist/expr/NewExpr.html)

`<init>` gets matched here:
https://github.com/imagej/ij1-patcher/blob/5f51a071752be66562861f9ee45077d44ffb8584/src/main/java/net/imagej/patcher/CodeHacker.java#L706-L712